history/games/insight.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
You need to clear cells before commiting this file.
This way we will not upload unnecessary/auto-generated data.